--- origsrc/activity-log-manager-0.9.4/data/Makefile.am	2012-02-22 13:24:08.000000000 -0600
+++ src/activity-log-manager-0.9.4/data/Makefile.am	2012-12-18 18:36:50.131088200 -0600
@@ -21,7 +21,7 @@ nobase_dist_data_DATA = \
 	icons/hicolor/48x48/apps/activity-log-manager.svg
 
 desktopdir = $(datadir)/applications
-desktop_in_files = activity-log-manager.desktop.in activity-log-manager-ccpanel.desktop.in
+desktop_in_files = activity-log-manager.desktop.in
 
 if HAVE_CCPANEL
 desktop_in_files+ = activity-log-manager-ccpanel.desktop.in
--- origsrc/activity-log-manager-0.9.4/src/Makefile.am	2012-03-02 09:31:27.000000000 -0600
+++ src/activity-log-manager-0.9.4/src/Makefile.am	2012-12-18 18:33:16.888891400 -0600
@@ -4,9 +4,10 @@ ccpanel_LTLIBRARIES = libactivity-log-ma
 endif
 if HAVE_WHOOPSIE
 gnomeccuidir = $(datadir)/gnome-control-center/ui/
+SUBDIRS = diagnostics
 endif
 
-SUBDIRS = diagnostics
+DIST_SUBDIRS = diagnostics
 
 VALAFLAGS = \
 	--pkg zeitgeist-1.0 \
